_posts/2019-07-12-service-design-anti-patterns.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: post
+title: Service Design Anti-patterns
+author: sam_quayle
+---
+
+![A platform at a tube station which says 'Mind the gap'!]({{ "/images/mind_the_gap.jpg"}})
+
+Despite its recent heightened profile – thanks in no small part to the efforts of the UK’s Government Digital Services – the field of Service Design is not as new as you might think. According to [this article](https://service-design.co/book-a-tiny-history-of-service-design-368ed603797c), you can trace its origins back to the turn of the 20th century with Taylorism and Human Factors Engineering through to the 1980s when Lynn Shostack first coined the term ‘Service Design’ and it began to emerge as a practice in its own right. Since then, there has been no shortage of articles, books, talks and consultancies describing the benefits, best practices and success stories of this burgeoning field. But what about the failures? What are the pitfalls, the misconceptions, the anti-patterns that can scupper even the most well-intentioned teams?
+
+Here are the most common mistakes I’ve encountered (and made), over the last five years delivering services, and how to avoid them:
+
+# Unempowered Service Managers
+Turning service designing thinking from a plan into implementation takes a lot of work. Often, it requires orchestrating change across a range of areas within an organisation: aligning teams, processes and communications. With this in mind, the person responsible for delivering a service needs to have the knowledge, relationships and authority to make the big decisions required to catalyse the required change. A Product Owner, alone, won’t cut it. Product Owners should be left to do what they do best: Make timely decisions, set priorities and represent the users throughout the delivery process. An experienced, well-connected Service Manager can not only provide support and escalation for Products Owners but can often be the difference between delivering a cohesive service and disjointed one. Or worse, not delivering a service at all. 
```

Comment:
Should it be _Turning service **design** thinking_? I don't think you want the commas in _A Product Owner, alone, won’t cut it._ as they suggest _alone_ is an optional word, which it isn't. I think it reads fine without them and is clearer.